packages/terra-dropdown-button/translations/de.json

```diff
@@ -1,4 +1,5 @@
 {
   "Terra.dropdownButton.moreOptions": "Weitere Optionen",
-  "Terra.dropdownButton.selected": "Ausgewählt"
+  "Terra.dropdownButton.selected": "Ausgewählt",
+  "Terra.dropdownButton.of": "von"
 }
```

Comment:
Already have this translations in other component, got it from there.

Reply:
Can you link the component?

Reply:
Since we already have a translations jira, let's only update the english locales and have the translations team provide the translations and update the other locales in one go just to be safe and not have to fix these later.